app/commands/beers.rb

```diff
@@ -8,10 +8,13 @@ def handle_call(message)
       chat_id = message.chat.id
 
       if beer_service.last(chat_id) && beer_service.drinks_fast?(chat_id)
-        message = ['*Easy, you*', '*Who drives you home*?', '*What are you, Homer Simpson?*'].sample
+        message = <<~MARKDOWN
+          ⏳ Try to do it after #{next_drink_adviced_time(chat_id)} (We have a timeout 5 minutes between glasses) ⌛
```

Comment:
How about?:

⏳ I don't believe you are drinking that fast ⏳
(we have a timeout 5 minutes between glasses, try again after 21:22:16)
Already drunk: 🍻🍻🍻

Reply:
I agree, it will be better